server.js

```diff
@@ -16,11 +39,61 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+
+// ROUTES
+  app.get("/", (req, res) => {
+  const HappyThoughtsAPI = {
+    message: "Welcome to Our Happy Thoughts-API! Go to /thoughts to see the 20 most recently posted happy thoughts",
+    frontend: "See live frontend site at https://our-happy-thoughts.netlify.app/ 🌞",
+  };
```

Comment:
Really nice welcoming page which made me smile! Maybe that you put the " before the sun to avoid confusion